index.html

```diff
@@ -1,21 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width,initial-scale=1" />
 
-    <link rel="icon" href="/favicon.png" />
-    <title>Manager</title>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width,initial-scale=1" />
 
-    <script type="module" src="/src/main.ts"></script>
-  </head>
-  <body>
-    <noscript>
-      <strong>
-        We're sorry but this site doesn't work properly without JavaScript enabled. Please enable it to continue.
-      </strong>
-    </noscript>
+  <link rel="icon" href="/favicon.png" />
+  <title>Manager</title>
+
+  <script type="application/json" id="kuma-config">{{.}}</script>
```

Comment:
So in dev you rely on something different?

Reply:
We’d do something like this: When building for production, only the if branch code will remain.

```js
if (import.meta.env.PROD) {
  const pathConfig = JSON.parse(document.querySelector('#kuma-config')?.textContent)
  console.log(pathConfig.guiBasePath)
} else {
  // Read some mock data during development
}
```